its deeeeebugged

SeyedMostafaHosseinian opened the pull request.

SeyedMostafaHosseinian committed:
its deeeeebugged

mostafa8026 commented on session-04-class/src/index.ts:
```diff
@@ -3,7 +3,7 @@ import { kindFeature } from './rtk/feature/kind/kind-slice';
 
 store.subscribe(()=>{
-    console.log(store.getState());
+    // console.log(store.getState());
```
Add some comments.

mostafa8026 commented on session-04-class/src/rtk/feature/kind/kind-slice.ts:
```diff
@@ -50,26 +52,33 @@ const slice = createSlice({
+                payload: OrderInterface,
+                type: string
+            }) => {
+                action.payload.items.map(item => {
```
use above function here

SeyedMostafaHosseinian committed:
add gitignore

SeyedMostafaHosseinian committed:
Create README.md

SeyedMostafaHosseinian committed:
Merge branch 'master' of https://github.com/mostafa8026/angular-course

SeyedMostafaHosseinian committed:
Changes to be committed:
	modified:   session-04-class/README.md
	modified:   session-04-class/src/rtk/app/store.ts
	modified:   session-04-class/src/rtk/feature/kind/kind-slice.ts
